pelicanconf.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*- #
+from __future__ import unicode_literals
+
+AUTHORS = (
+    u'The Kinto team',
+)
+
+SITENAME = u'Kinto Storage'
+SITEURL = 'http://localhost:8000'
+
+PATH = 'content'
+
+TIMEZONE = 'Europe/Paris'
+
+DEFAULT_LANG = u'fr'
+
+DEFAULT_PAGINATION = False
+
+THEME = "theme"
+INDEX_SAVE_AS = 'news.html'
```

Comment:
This is how the `news.html` page looks like:

![](http://i.imgur.com/OxsD3b8.png)

Would be great to have a sample article and proper layout